### PR TITLE
Including this and base constructor calls in RequireNamedArgumentsAna…

### PR DESCRIPTION
**Pull request by JeffAshton**
…lyzer

### CODE REVIEW

File: src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs [hunk 1]
```diff
@@ -49,26 +47,35 @@ ArgumentSyntax syntax
 				AnalyzeCallSyntax,
 				SyntaxKind.ObjectCreationExpression
 			);
+
```
Comment (JeffAshton) [inline src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs:50]: https://www.youtube.com/watch?v=6iCPIUGnHQ8

File: src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs [hunk 2]
```diff
@@ -89,22 +96,22 @@ SyntaxNodeAnalysisContext ctx
 				ctx.ReportDiagnostic(
 					Diagnostic.Create(
 						descriptor: Diagnostics.TooManyUnnamedArgs,
-						location: expr.GetLocation(),
+						location: ctx.Node.GetLocation(),
```
Comment (JeffAshton) [inline src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs:99]: `expr` was `(ExpressionSyntax)ctx.Node` previously

**Comment by j3parker** [comment]
This is a good fix, but be prepared for a bit of backlog rolling this out. It might make sense to roll this change out on its own.
(cases where more than 5 args used in `this`/`base`)

You can use the code fix and "fix all in solution" to fix those. I often use `Lms.sln` to do it all at once but that thing seems to only be usable every-other VS release 🤷

**Comment by JeffAshton** [comment]
> This is a good fix, but be prepared for a bit of backlog rolling this out. It might make sense to roll this change out on its own.
> (cases where more than 5 args used in `this`/`base`)
> 
> You can use the code fix and "fix all in solution" to fix those. I often use `Lms.sln` to do it all at once but that thing seems to only be usable every-other VS release 🤷

I got a successful build with only fixing 2 spots:

https://git.dev.d2l/projects/CORE/repos/lms/pull-requests/12981/diff

**Pull request merged**